versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/AbstractDependencyUpdatesReport.java

```diff
@@ -80,6 +81,16 @@ public abstract class AbstractDependencyUpdatesReport extends AbstractVersionsRe
     @Parameter(property = "processDependencyManagementTransitive", defaultValue = "true")
     protected boolean processDependencyManagementTransitive;
 
+    /**
+     * <p>Include dependencies with version set in a parent or in a BOM.</p>
+     * <p>This is similar to {@code processDependencyManagementTransitive}, but will
+     * report updates on dependencies.</p>
+     *
+     * @since 2.18.1
```

Comment:
Maybe should be a 2.19.0 as we add new feature ... what do you think?

Reply:
or it is for fixing bug only ...

Reply:
Ok. Changed that in DisplayDependencyUpdatesMojo as well.

Reply:
Well, a bug fix and a feature toggle. I don't know :-)